lib/swagcov/install.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Swagcov
+  class Install
+    def generate_dotfile
+      dotfile = Rails.root.join(".swagcov.yml").to_s
+
+      return if File.exist?(dotfile)
+
+      File.write(
+        Rails.root.join(dotfile).to_s,
```

Comment:
Two things:

- Since rails is a dependency, you could also consider using Thor to handle the copy of the template over
- Instead of the inline hash I think you might be able to create a `lib/swagcov/templates/.swagcov.yml` file and just copy it over (with Thor or fileutils)

Reply:
Ah ya, I forgot Thor came with rails.